dists/fs-repo-migrations/matrices/v1.0.0

```diff
@@ -0,0 +1,10 @@
+darwin 386
```

Comment:
ah, okay. I just picked a few that seemed reasonable, i'll remove it. But all of these binaries are supported by the go compiler. Which do you think should be kept?